OpenRA.Mods.Cnc/OpenRA.Mods.Cnc.csproj

```diff
@@ -3,7 +3,7 @@
     <TargetFramework>net472</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Optimize>true</Optimize>
-    <LangVersion>5</LangVersion>
+    <LangVersion>6</LangVersion>
```

Comment:
Please keep `<LangVersion>5</LangVersion>`.

Reply:
I've reverted this back to 5 and fixed the code - out of interest why keep the LangVersion to 5, is there a particular reason or is it mostly the code style is established and allowing different syntax would not help consistency?

Reply:
> code style is established and allowing different syntax would not help consistency?

Entirely this. The newer language versions bring some nice features that would benefit OpenRA, but also a bunch of oddities that wouldn't. A discussion needs to happen to revise the expected code style then all the projects should be bumped at the same time. There hasn't been a strong push to actually do this, though.